addons/account/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,387 @@
+??
+
+account      / res.company              / account_setup_bank_data_done (boolean): DEL
+account      / res.company              / account_setup_bank_data_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+account      / res.company              / account_setup_bar_closed (boolean): DEL 
+account      / res.company              / account_setup_coa_done (boolean): DEL 
+account      / res.company              / account_setup_coa_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+account      / res.company              / account_setup_company_data_done (boolean): DEL 
+account      / res.company              / account_setup_fy_data_done (boolean): DEL 
+account      / res.company              / account_setup_fy_data_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+# DONE: post-migration: filled the new *_state fields
+
+account      / res.company              / accounts_code_digits (integer): DEL
+account      / res.company              / incoterm_id (many2one)        : previously in module l10n_be_intrastat
+account      / res.company              / invoice_is_email (boolean)    : NEW 
+account      / res.company              / invoice_is_print (boolean)    : NEW 
+account      / res.company              / invoice_reference_type (selection): NEW selection_keys: function
+account      / res.company              / qr_code (boolean)             : NEW
+account      / res.company              / transfer_account_code_prefix (char): NEW 
+# NOTHING TO DO
+
+---XML records in module 'account'---
+DEL account.financial.report: account.account_financial_report_assets0 (noupdate)
+DEL account.financial.report: account.account_financial_report_balancesheet0 (noupdate)
+DEL account.financial.report: account.account_financial_report_expense0 (noupdate)
+DEL account.financial.report: account.account_financial_report_income0 (noupdate)
+DEL account.financial.report: account.account_financial_report_liability0 (noupdate)
+DEL account.financial.report: account.account_financial_report_liabilitysum0 (noupdate)
+DEL account.financial.report: account.account_financial_report_profitandloss0 (noupdate)
+DEL account.financial.report: account.account_financial_report_profitloss_toreport0 (noupdate)
+NEW account.incoterms: account.incoterm_CFR
+NEW account.incoterms: account.incoterm_CIF
+NEW account.incoterms: account.incoterm_CIP
+NEW account.incoterms: account.incoterm_CPT
+NEW account.incoterms: account.incoterm_DAF
+NEW account.incoterms: account.incoterm_DAP
+NEW account.incoterms: account.incoterm_DAT
+NEW account.incoterms: account.incoterm_DDP
+NEW account.incoterms: account.incoterm_DDU
+NEW account.incoterms: account.incoterm_DEQ
+NEW account.incoterms: account.incoterm_DES
+NEW account.incoterms: account.incoterm_EXW
+NEW account.incoterms: account.incoterm_FAS
+NEW account.incoterms: account.incoterm_FCA
+NEW account.incoterms: account.incoterm_FOB
```

Comment:
We should rename these XML-IDs out of the `stock` ones

Reply:
But in `base` pre-migration is done the rename of the model

Reply:
But this is not model renaming, but XML-IDs renaming, or we will have duplicated records.

Reply:
Ahh true!!